Samples/HelloDirectML/main.cpp

```diff
@@ -260,9 +260,7 @@ int main()
     // The temporary resource is scratch memory (used internally by DirectML), whose contents you don't need to define.
     // The persistent resource is long-lived, and you need to initialize it using the IDMLOperatorInitializer.
 
-    UINT64 temporaryResourceSize = std::max(
-        initializeBindingProperties.TemporaryResourceSize,
-        executeBindingProperties.TemporaryResourceSize);
+    UINT64 temporaryResourceSize = initializeBindingProperties.TemporaryResourceSize;
```

Comment:
This doesn't look right. I believe the intention was to share a single buffer for both initialization and execution, and that buffer should be large enough for both. There is a call to `CloseExecuteResetWait` between init and execute to make this possible. 

The proposed change could result in too small a temporary buffer for execute (or even 0, so no resource even exists).